tests/unit/plugins/test_writer.py

```diff
@@ -239,6 +250,10 @@ def test_pipeline_generation(
     file_names = [pipeline_name + file_suffix for file_suffix in file_endings]
     if framework == "DVC":
         file_names.append("dvc.yaml")
+        if pipeline_name == "dvc_pipeline_a_b0_stageperartifact":
```

Comment:
lets drop this if statement so this test will run for other tests in the future too.

Reply:
Fixed